packages/js/product-editor/src/blocks/generic/linked-product-list/edit.tsx

```diff
@@ -112,8 +124,46 @@ export function Edit( {
 		setLinkedProductIds( newLinkedProductIds );
 	}
 
+	async function choseProductsForMe() {
+		const { getRelatedProducts } = resolveSelect( PRODUCTS_STORE_NAME );
+		dispatch( {
+			type: 'LOADING_LINKED_PRODUCTS',
+			payload: {
+				isLoading: true,
+			},
+		} );
+
+		const relatedProducts = ( await getRelatedProducts(
+			productId
+		) ) as Product[];
```

Comment:
I did notice one issue with this, for saving the product we use the core entity data store, but for this selector we are using our product data store. This means that when a user save the product by adding categories and then clicks `getRelatedProducts` we get the same result instead of the updated one's.

Should we either make use of the core entity data store as well? or add an invalidation method when we save the product, so we also invalidate our products data store?

Reply:
👁️ looking

Reply:
I don't know if we can use the core store. Is the related prop (`related_ids`) exposes by code? Maybe we can tweak the selector deps to invalidate its data.

Reply:
You should be able to use `useEntityProp` and get the `related_ids` that way